dart/lib/src/protocol/breadcrumb.dart

```diff
@@ -76,6 +76,21 @@ class Breadcrumb {
   /// The value is submitted to Sentry with second precision.
   final DateTime timestamp;
 
+  /// Creates a [Breadcrumb] from deserialized JSON [Map].
+  factory Breadcrumb.fromJson(Map<String, dynamic> json) {
+    final levelName = json['level'];
+    return Breadcrumb(
+      timestamp: json['timestamp'] != null
+          ? DateTime.tryParse(json['timestamp'])
+          : null,
```

Comment:
accessing multiple times a Map per key has the downside of multiple lookups
I'd cache ``json['timestamp'] to a var and use it twice